ansible/playbooks/ubuntu.yml

```diff
@@ -33,6 +33,13 @@
       apt_repository: repo='deb http://repos.azulsystems.com/ubuntu stable main'
       tags: patch_update
 
+    - name: Add the azul key
+      apt_key:
```

Comment:
:exclamation: Secrets alert! Is there some way to store this info on vault or some secret place and read it off from there? Does ansible provide such facility?